radioactivedecay/inventory.py

```diff
@@ -200,49 +454,55 @@ def wrapper(*args, **kw):
 
 class Inventory:
     """
-    ``Inventory`` instances store a dictionary of radionuclides and associated activities, and a
+    ``Inventory`` instances store a dictionary of radionuclides and associated numbers, and a
     ``DecayData`` instance of radioactive decay data.
 
     Parameters
     ----------
     contents : dict
-        Dictionary containing radionuclide strings or Radionuclide objects as keys and activities
+        Dictionary containing radionuclide strings or Radionuclide objects as keys and numbers
         as values.
+    input_type : str, optional
+        Type of input (masses, numbers, activities; activities is default)
     check : bool, optional
         Check for the validity of contents (default is True).
     data : DecayData, optional
         Decay dataset (default is the ICRP-107 dataset).
 
     Attributes
     ----------
-    contents : dict
-        Dictionary containing radionuclide strings as keys and activities as values. Radionuclides
+    numbers : dict
+        Dictionary containing radionuclide strings as keys and numbers as values. Radionuclides
```

Comment:
"radionuclide strings" -> "nuclide string" and "numbers" -> "number of nuclides"